fixture-monkey-datafaker/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id("org.jetbrains.kotlin.jvm")
+    id("com.navercorp.fixturemonkey.gradle.plugin.java-conventions")
+    id("com.navercorp.fixturemonkey.gradle.plugin.maven-publish-conventions")
+}
+
+dependencies {
+    api(project(":fixture-monkey-api"))
+    implementation("net.datafaker:datafaker:1.9.0")
```

Comment:
NIT, it is better to specfiy in version catalog.

Reply:
I've fixed this in commit 7afcd18.